### ISSUE TITLE
slugcomment at top-right of title page needs \noindent

### ISSUE THREAD

**Issue by davidwhogg** [AASJournals/AASTeX60#13]
It doesn't align with the left margin!

**Comment by augustfly** [comment]
for that matter the affiliation notemarks float into the side margin which just can't be stylistically correct. marking as a bug.

**Comment by augustfly** [comment]
downstream clarification. I believe @davidwhogg means "slugcomment at top-**left** of title page" since I'm unable to create any slugcomment on the top right of the title page.

**Comment by davidwhogg** [comment]
Yes I meant top-left. Sorry, still a mess when it comes to the whole left-right thing.

And yes, it needs the \noindent.

**Comment by davidwhogg** [comment]
ps. I fixed the code with this:
```
\makeatletter % you know you are living your life wrong when you need to do this                                                      
\long\def\frontmatter@title@above{
\vspace*{-\headsep}\vspace*{\headheight}
\noindent\footnotesize
{\noindent\footnotesize\textsc{\@journalinfo}}\par
{\noindent\scriptsize Preprint typeset using \LaTeX\ style AASTeX6 v.\ 1.0 but heavily modified by David W. Hogg (NYU).
}\par\vspace*{-\baselineskip}\vspace*{0.625in}
}%                                                                                                                                    
\makeatother
```

**Comment by augustfly** [comment]
always appreciated.

**Comment by augustfly** [comment]
as are these fixes which have been forwarded to the latex contractor...

https://github.com/dfm/peerless/blob/master/document/ms.tex#L19-L69

**Comment by jradavenport** [comment]
👍 these mods are 💯

**Comment by gregschwarz** [comment]
The \slugcomment has been removed in v6.1. Instead we have the following:

\received{receipt date}
\revised{revision date}
\accepted{acceptance date}
\published{published date}
\submitjournal{AASJournal name}

These commands will appear after the affiliation information and before the abstract. The text "Submitted to" is appended to the AASJournal name argument in the \submitjournal command.

http://journals.aas.org/authors/aastex/aasguide61.html#editorial

**Comment by augustfly** [comment]
@davidwhogg Is this closed with the incorporation of `modern` in `v6.1`? Or are there other outstanding issues we can address in v6.2?

**Comment by davidwhogg** [comment]
closing...
